cost/kubecost/sizing/kubecost_resizing_recommendation.pt

```diff
@@ -0,0 +1,249 @@
+name "Kubecost Request Right Sizing Recommendations"
+rs_pt_ver 20180301
+type "policy"
+short_description "Pulls the Request Sizing Recommendations from Kubecost and raises and incident. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/kubecost/sizing/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description ""
+severity "low"
+category "Cost"
+default_frequency "daily"
+tenancy "single"
+info(
+  version: "0.1",
+  provider: "Kubecost",
+  service: "Kubernetes",
+  policy_set: ""
+)
+
+###############################################################################
+# User inputs
+###############################################################################
+
+parameter "param_email" do
+  label "Email addresses"
+  description "Email addresses of the recipients you wish to notify"
+  type "list"
+end
+
+parameter "param_kubecost_host" do
+  type "string"
+  label "Kubecost Host or IP address"
+  description "IP Address of Kubecost LB"
+end
+
+parameter "param_window" do
+  type "string"
+  label "Range to analyze"
+  description "Accepts duration in number of days, hours or minutes e.g., 2d, 30m, 6h"
+  default "2d"
+end
+
+parameter "param_cpu_algo" do
+  type "string"
+  label "The algorithm for CPU"
+  description "The algorithm to be used to calculate CPU recommendations"
```

Comment:
Defines whether max or a certain quantile should be used  to compute CPU request recommendation